### PR TITLE
Split checks into hypernode and application specific

### PR DESCRIPTION
**Pull request by hrithikkothari1234**
Split checks into hypernode and application specific

### CODE REVIEW

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md [hunk 1]
```diff
@@ -57,16 +57,19 @@ Horizontal autoscaling is available on all the Falcon cloud plans (OpenStack).
...
 
-### Operating system
+#### Operating system
 
 - The operating system of the Hypernode should be Debian Bookworm
 
-### Enable and configure Varnish
+#### Make sure the Hypernode is a production plan
+
+Unfortunately we don't support Horizontal autoscaling for development plans.
```
Comment (tdgroot) [inline docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:70]:
```suggestion
For now, we don't support horizontal autoscaling for development plans.
```

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md [hunk 2]
```diff
@@ -160,9 +153,29 @@ After the version validation, please verify the MySQL host is set to `mysqlmaste
 
 You should see something similar to `'host' => 'mysqlmaster',`. If this is not the case please make sure the database connection host is set to `mysqlmaster` instead of `localhost` or `127.0.0.1` in the magento configuration file at `<magento_root>/app/etc/env.php`.
 
-### Make sure the Hypernode is a production plan
+### Application Specific Requirements - Magento2
 
-Unfortunately we don't support Horizontal autoscaling for development plans.
+#### Supported CMS
+
+Horizontal autoscaling is available for Magento2.
```
Comment (tdgroot) [inline docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:160]:
```suggestion
Horizontal autoscaling is available for Magento 2.4.7 and higher.
```

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md [hunk 3]
```diff
@@ -160,9 +153,29 @@ After the version validation, please verify the MySQL host is set to `mysqlmaste
...
+To make use of Horizontal autoscaling, there are a couple of other requirements the application should meet.
+
+#### Make sure Elasticsearch/Opensearch configured properly
+
+Please make sure Elasticsearch or Opensearch host is set to `elasticsearchmaster` in the Magento2 configuration file at `<magento_root>/app/etc/env.php`
+More information about [Elasticsearch o Hypernode](../../hypernode-platform/tools/how-to-use-elasticsearch-on-hypernode.md)
```
Comment (tdgroot) [inline docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:166]:
```suggestion
More information about [Elasticsearch on Hypernode](../../hypernode-platform/tools/how-to-use-elasticsearch-on-hypernode.md)
```

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md [hunk 4]
```diff
@@ -160,9 +153,29 @@ After the version validation, please verify the MySQL host is set to `mysqlmaste
...
-### Make sure the Hypernode is a production plan
+### Application Specific Requirements - Magento2
 
-Unfortunately we don't support Horizontal autoscaling for development plans.
+#### Supported CMS
+
+Horizontal autoscaling is available for Magento2.
+To make use of Horizontal autoscaling, there are a couple of other requirements the application should meet.
+
+#### Make sure Elasticsearch/Opensearch configured properly
```
Comment (tdgroot) [inline docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:163]:
```suggestion
#### Make sure Elasticsearch/Opensearch is configured properly
```

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md [hunk 5]
```diff
@@ -160,9 +153,29 @@ After the version validation, please verify the MySQL host is set to `mysqlmaste
...
+
+Please make sure Elasticsearch or Opensearch host is set to `elasticsearchmaster` in the Magento2 configuration file at `<magento_root>/app/etc/env.php`
+More information about [Elasticsearch o Hypernode](../../hypernode-platform/tools/how-to-use-elasticsearch-on-hypernode.md)
+
+#### Make sure RabbitMQ configured properly
+
+Please make sure RabbitMQ host is set to `rabbitmqmaster` in the Magento2 configuration file at `<magento_root>/app/etc/env.php`
+More information about [RabbitMQ o Hypernode](../../best-practices/database/how-to-run-rabbitmq-on-hypernode.md)
+Also make sure to configured RabbitMQ without guest user.
+
```
Comment (tdgroot) [inline docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:173]: Either here or in the rabbitmq documentation page it should be clear that there's a non-default rabbitmq provisioned by default, called `hypernode-admin`, but users are also free to create new users if they're more comfortable with that.

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md [hunk 6]
```diff
@@ -160,9 +153,29 @@ After the version validation, please verify the MySQL host is set to `mysqlmaste
...
+#### Make sure Database storage is disabled & Remote storage is enabled and configured.
+
+Please make sure to enable remote storage for your application and configure it correctly as only AWS-s3 remote storage drivers are supported.
```
Comment (tdgroot) [inline docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:176]: For now, link to the magento documentation: https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/storage/remote-storage/remote-storage-aws-s3

But later on we should have our own documentation on this!

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md (hunk 5 above)
Comment (hrithikkothari1234) [reply docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:173]: Added the info to both the docs. The more the merrier 🎅

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md (hunk 6 above)
Comment (hrithikkothari1234) [reply docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:176]: Agreed, thanks! I put the link for reference

**Review by tdgroot — APPROVED** [review]

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md [hunk 7]
```diff
@@ -57,16 +57,19 @@ Horizontal autoscaling is available on all the Falcon cloud plans (OpenStack).
...
+We will divide them between Hypernode-specific and Application-specific requirements
 
-Horizontal autoscaling is available for Magento2.
-To make use of Horizontal autoscaling, there are a couple of other requirements the Hypernode and the application should meet:
+### Hypernode Specific Requirements
 
-### Operating system
+#### Operating system
 
 - The operating system of the Hypernode should be Debian Bookworm
```
Comment (martijneichelsheim) [inline docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:66]: Probably point them to support if they want to upgrade their hypernode

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md [hunk 8]
```diff
@@ -104,7 +107,7 @@ acl purge {
 }
 ```
 
-### Enable and configure Redis Persistent
+#### Enable and configure Redis Persistent
```
Comment (martijneichelsheim) [inline docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md:110]: Redis + redis persistent is also Application specific

Comment (hrithikkothari1234) [reply docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md]: Moved to application-specific requirements

File: docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md (hunk 7 above)
Comment (hrithikkothari1234) [reply docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md]: Added link to customer support

**Review by martijneichelsheim — APPROVED** [review]

**Pull request merged**